### PR TITLE
Mlitkey/build test action

### PR DESCRIPTION
**Pull request by mlitkey**
Replace existing Travis CI pipeline with Github Action to be consistent with release pipeline

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: CI
+on: push
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```
Comment (briantist) [inline .github/workflows/ci.yml]: Why not add 3.9 too 👀?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: CI
+on: push
```
Comment (briantist) [inline .github/workflows/ci.yml]:
```suggestion
on:
  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
  push:
  pull_request:
```

We want to run on PRs that come from forks too. Push will catch pushes to branches on this project, and merging of PRs, but not the PR itself.

**Review by briantist — CHANGES_REQUESTED** [review]
This looks good to me and the tests all seem to run and pass. Only blocking addition I'm asking for is the PR trigger.

Other than that I definitely want to get @zturechek or @chintanms 's input as I have less familiarity with the project itself; I really just worked on the GHA part

**Comment by mlitkey** [comment]
@briantist Do you want me to remove the travis config at the same time or leave that for a bit until we are certain this is working how we want? They are functionally equivalent as far as I can tell.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mlitkey) [reply .github/workflows/ci.yml]: Done!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (mlitkey) [reply .github/workflows/ci.yml]: Makes sense. Done!

**Comment by briantist** [comment]
> @briantist Do you want me to remove the travis config at the same time or leave that for a bit until we are certain this is working how we want? They are functionally equivalent as far as I can tell.

I'm gonna leave that @zturechek I think.. unfortunately I have no context on the Travis tests. My naive opinion would be to remove them, but I don't want to say authoritatively for this one.

**Review by briantist — APPROVED** [review]
Looks great from me, will final call to Zak, thanks @mlitkey 🙂

**Review by zturechek — APPROVED** [review]
lgtm too @mlitkey. if its functioning the same from your testing, i say rip the bandaid off and remove the travis CI config

**Pull request merged**